Update README.md

Vaishnavi798 opened the pull request:
Added contribution guidelines and made certain changes in the readme file.

Vaishnavi798 committed:
Update README.md

Added contribution guidelines and made certain changes in the readme file.

Vaishnavi798 committed:
Update README.md

kothariji commented on README.md:
```diff
@@ -41,37 +45,61 @@
 ![4](https://contributors-img.web.app/image?repo=kothariji/syntaxmeets)
 
 
-### If you liked our work do give us a star :star::star::star: It Encourages us to do more :wink: :dizzy:
```
We need wink and dizzy emojis!!

kothariji commented on README.md:
```diff
@@ -41,37 +45,61 @@
+
+## Contributors:
 
-### Thank you for Contributing :pray: :dizzy:
```
We need pray emoji

kothariji commented on README.md:
```diff
@@ -41,37 +45,61 @@
+## Contribution Guidelines:<br>
+-To contribute to this project, go through the step-by-step guidelines provided [here.](https://github.com/kothariji/SyntaxMeets/blob/master/CONTRIBUTING.md)
+Contributors should strictly abide by the following points while contributing:<br>
+- Make sure you do not copy code from external sources because that work will not be considered.<br>
+- Plagiarism is strictly not allowed.<br>
```
Can you please explain what do you mean by plagiarism here  ? As this repository is not the collection of algorithms or similar thing, where plagiarism matters.

Vaishnavi798 replied:
With plagiarism, it is meant to say that while contributing, one should not copy the ideas or the code of any other project similar to this project. There are many other projects with similar idealogy with this project therefore we should have some uniqueness which would differentiate them so, to make sure that this uniqueness is maintained it is necessary that while contributing one should make sure that the ideas proposed by him/her are unique and their own and not copied from any other source.

kothariji replied:
Cool! I agree with this point 🙌

kothariji commented on README.md:
````diff
@@ -41,37 +45,61 @@
 
-### To run the server, go to the server folder in the backend directory and Run 
+- <b>Add a Remote the Original Project Repository: </b><br>
+Remote means the remote location of project on Github.This will point to the original repository. It will help you to get the latest changes made by others.
+```
+$ cd <your-forked-project-folder>
+$ git remote add upstream https://github.com/<author-account-username>/<project>.git
+``` 
````
Don't you think, giving details about how **remote** works in the readme is good practice?

Vaishnavi798 replied:
I think, giving details about how remote works in the readme is good practice because if we consider a beginner who wants to contribute to the project but have very less or no knowledge of git ,it would become easier for him/her to make his contributions effectively if he/she gets all the info in the project's readme itself and does not use/require any other source for making contributions to the project. So, it would be effective if we provide a short info about remote in the readme itself, taking a beginner under consideration.

kothariji replied:
Yes, I agree with you, So these are contributing guidelines right. So IMO it should come under Contributing.md and not readme.md, What Say?

Vaishnavi798 replied:
Yes, you are right, these should come under Contributing. md. But, I think that adding these guidelines in the readme. MD would help the contributors get a quick glimpse of how they can contribute by just reading the readme. md file only. But still, if you suggest I'll make amendments accordingly to Contributing. md file.

kothariji commented on README.md:
```diff
@@ -4,19 +4,23 @@
 [
 ![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues) [![Open Source? Yes!](https://badgen.net/badge/Open%20Source%20%3F/Yes%21/blue?icon=github)](https://github.com/kothariji/SyntaxMeets) 
 
-[![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://github.com/kothariji/SyntaxMeets) [![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/kothariji/SyntaxMeets)
```
We need the Build with Love Badge

kothariji reviewed (changes requested):
I would request you to see all the requested changes, and modify accordingly

Vaishnavi798 committed:
Updated changes to Readme.md

Updated contribution guidelines by adding a link to the CONTRIBUTING.md file for the ease of the viewers and certain guidelines are mentioned regarding contribution to the project.

Vaishnavi798 committed:
Updated readme with requested changes

Vaishnavi798 commented:
I have made the changes according to the description I've provided above. If there's still room for improvement, please let me know.
